src/index.html

```diff
@@ -28,38 +28,38 @@
     <div class="cover-container d-flex h-100 mx-auto flex-column">
         <header class="masthead">
             <div class="inner">
-                <h3 class="masthead-brand rainbowText">Super Mario 64 JS</h3>
                 <nav class="nav nav-masthead justify-content-center">
-                    <!--a class="nav-link" href="https://server.sm64js.com/createGame">Custom Game</a-->
+                    <a class="nav-link" href="https://server.sm64js.com/createGame">Custom Game</a>
```

Comment:
we should not display this link, because it's not working and will most likely not come back anytime soon